src/components/CreateAccount.vue

```diff
@@ -156,7 +158,7 @@
             >
               <div class="flex items-center w-full xs:w-auto">
                 <!-- This is for emit method -->
-                <Button type="submit" @emitclick="consoleClick">
+                <Button type="submit" @emitclick="consoleClick" :btn-primary="true">
```

Comment:
`emitclick` should be `click`

Reply:
@giteshk11 I got the issues. I will fix it and let you know.